### DISCUSSION TITLE
Plugin is not listed in Community Plugins

### DISCUSSION THREAD

**Discussion by gssln** [hadynz/obsidian-kindle-plugin#240] [Q&A]
I just started using Obsidian and wanted to sync my Kindle notes. However, this plugin seems to have disappeared from the community plugins in Obsidian. Any reason/update/workaround?

**Comment by wilbr11** [comment]
I'm having the same problems

**Comment by wilbr11** [comment]
I just read that Obsidian has delisted it because it was sharing data.  Apparently the dev was notified and is fixing it, but it hasn't been relisted
